lib/integrations/slack.rb

```diff
@@ -46,11 +46,11 @@ def message_color
     private
 
     def message_text
+      description = run[:description].nil? ? '' : "(#{run[:description]}) "
       {
-
-        'run_completion' => "Your Rainforest Run <#{payload[:frontend_url]}|##{payload[:id]}> #{run[:status]}.",
-        'run_error' => "Your Rainforest Run <#{payload[:frontend_url]}|##{payload[:id]}> errored: #{run[:error_reason]}.",
-        'run_webhook_timeout' => "Your Rainforest run <#{payload[:frontend_url]}|##{payload[:id]}> timed out due to your webhook failing. If you need a hand debugging it, please let us know via email at team@rainforestqa.com.",
+        'run_completion' => "Your Rainforest Run <#{payload[:frontend_url]}|##{payload[:id]}> #{description}#{run[:status]}.",
```

Comment:
Nope, I've added it in https://github.com/rainforestapp/rainforest-integrations/pull/23/files#diff-1981ddaa8f271ef1cdef12f13ba34863R49

Maybe I should make that more obvious though...